lib/rpc.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { ReceiverOptions, SenderOptions, createSession, createSender, createReceiver } from "./rhea-promise";
+import * as debugModule from "debug";
+const debug = debugModule("azure:event-hubs:rpc");
```

Comment:
what does rpc mean in this context?

Reply:
The module can be used to create new request/response or request or response links. Named it rpc because it deals with creating amqp constructs on top of a connection.